docs/using-kontena/grids.md

```diff
@@ -20,7 +20,9 @@ The [Grid](../core-concepts/architecture.md#the-grid) is top-level object in Kon
 $ kontena grid create --initial-size=3 mygrid
 ```
 
-Creates a new grid named `mygrid` with initial size of 3 nodes (grid must have at least 3 nodes that are part of etcd cluster).
+Creates a new grid named `mygrid` with initial size of 3 nodes (grid must have at least 3 nodes that are part of etcd cluster). Etcd member nodes create the actual (data) cluster and rest of the grid nodes act as `proxy` nodes for etcd. Typically you want to spread your grid nodes in different availability zones / regions so that losing majority of the initial nodes becomes virtually impossible. Losing majority impairs the etcd and will cause troubles within the Kontena grid for example in service load balancing.
```

Comment:
Probably not good idea to spread etcd across regions because latency... should we just recommend different az's here?